config/jobs/kubernetes-sigs/descheduler/OWNERS

```diff
@@ -0,0 +1,5 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- ingvagabund
+- damemi
```

Comment:
@seanmalloy do you want to be approver as well?

Reply:
Might want to add @aveshagarwal to the list of approvers

Reply:
@aveshagarwal do you want to be approver?